Tests/conf.json

```diff
@@ -3857,4 +3857,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
this is by mistake, right?

Reply:
Yes. Earlier, I was experiencing conflict in this file while merging the code. As suggested by Kobbi, I copied the content from master repo to my local machine and then pushed the code and it works fine.